{
	"101": {
		"0405": {
			"name": "PYTHON_CORE_CALL_FAILURE",
			"message": "",
			"short_description": "Unknown cause.",
			"long_description": "Unknown cause.",
			"short_solution": "Unknown solution.",
			"long_solution": "Unknown solution.",
			"support_link": "https://preten.do/101-0405"
		}
	}
}